spring-boot-sample-project/src/main/java/com/example/demo/service/UserService.java

```diff
@@ -0,0 +1,58 @@
+package com.example.demo.service;
+
+import com.example.demo.pojo.User;
+import com.example.demo.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+
+@Component
+public class UserService {
+
+
+    private UserRepository userRepository = new UserRepository();
+
+    public List<User> getAllUser(){
+        List<User> userList = userRepository.getUserList();
+        return userList;
+    }
+
+    public User getAUser(String name){
```

Comment:
`getAUser` is not a good method name. In fact, `User getUser(String name)` captures the intent well that the method will get one user with the supplied param